scripts/rollup/build.js

```diff
@@ -435,6 +435,8 @@ rimraf('build', async () => {
   try {
     // create a new build directory
     fs.mkdirSync('build');
+    // create the .tmp folder for local npm packing and unpacking
+    fs.mkdirSync(path.join('build', '.tmp'));
```

Comment:
We should delete this folder after the build.

Reply:
Fixed, together with the swap to `os.tmpdir()`